### PR TITLE
Plone6

### PR DESCRIPTION
**Pull request by ale-rt**

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -39,15 +41,17 @@ jobs:
...
           sudo locale-gen en_US.UTF-8 nl_NL@euro
           python -m venv .
+          ln -s requirements-${{ matrix.config[2] }}.txt requirements.txt
           bin/pip install -r requirements.txt
+          ln -s versions-${{ matrix.config[2] }}.cfg versions.cfg
           bin/buildout -c .github.cfg
       - name: Pre commit
         uses: pre-commit/action@v2.0.0
-        if: ${{ matrix.config[1] == '3.8' }}
+        if: ${{ matrix.config[1] == '3.11' }}
```
Comment (thet) [inline .github/workflows/main.yml:50]: 3.11 is not in the matrix.

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -39,15 +41,17 @@ jobs:
...
       - name: Code Analysis
         run: |
           bin/code-analysis
-        if: ${{ matrix.config[1] == '3.8' }}
+        if: ${{ matrix.config[1] == '3.11' }}
```
Comment (thet) [inline .github/workflows/main.yml:54]: same, 3.11 is not in the matrix.

File: src/euphorie/content/risk.py [hunk 3]
```diff
@@ -660,7 +659,6 @@ def fixed_priority(self, value):
         self.default_priority = value
 
     @property
-    @memoize
```
Comment (thet) [inline src/euphorie/content/risk.py:663]: I'm fine with using memoize sparsely and keep this change, but what problem does this solve?
Seems unrelated to the Plone 6 changes here.

**Review by thet — CHANGES_REQUESTED** [review]

File: .github/workflows/main.yml (hunk 1 above)
Comment (ale-rt) [reply .github/workflows/main.yml:50]: The 3.11 is planned but commented some lines above

File: .github/workflows/main.yml (hunk 2 above)
Comment (ale-rt) [reply .github/workflows/main.yml:54]: The 3.11 is planned but commented some lines above

**Review by thet — APPROVED** [review]
